apply/apply_test.go

```diff
@@ -224,49 +224,38 @@ func TestApplySmokeTest(t *testing.T) {
 	r.NoError(err)
 	// defer os.RemoveAll(d)
 
-	json := `
-{
-  "defaults": {
-    "aws_region_provider": "reg",
-    "aws_region_backend": "reg",
-    "aws_profile_provider": "prof",
-    "aws_profile_backend": "prof",
-    "aws_provider_version": "0.12.0",
-    "account_id": 789,
-    "infra_s3_bucket": "buck",
-    "project": "proj",
-    "terraform_version": "0.100.0",
-    "owner": "foo@example.com"
-  },
-  "travis_ci": {
-	"enabled": true,
-	"aws_iam_role_name": "travis",
-        "id_account_name": "id",
-        "test_buckets": 7
-  },
-  "accounts": {
-    "foo": {
-      "account_id": 123
-    },
-    "bar": {
-      "account_id": 456
-    }
-  },
-  "modules": {
-    "my_module": {}
-  },
-  "envs": {
-    "staging":{
-        "components": {
-            "comp1": {},
-            "comp2": {}
-        }
-    },
-    "prod": {}
-  }
-}
+	yaml := `
```

Comment:
As discussed IRL– we should just duplicate all the test methods that have json and create yaml equivalents. Once we are ready to remove json support we can then delete the json test methods.

Reply:
Also this test is using version 1, we should only worry about yaml support for v2.